src/main/resources/migrations.xml

```diff
@@ -3142,4 +3142,10 @@
 		</sql>
 	</changeSet>
 
+	<changeSet id="delete-peoplePositions-without-position-and-person" author="sahindoganlutfi">
+		<comment>Cleanup the NULL of the personUuid and positionUuid columns of the peoplePositions table.</comment>
+		<sql>
+			DELETE FROM "peoplePositions" WHERE "personUuid" IS NULL and "positionUuid" IS NULL;
+		</sql>
+	</changeSet>
 </databaseChangeLog>
```

Comment:
Please keep the blank line after the last changeset.

Reply:
Added blank line migrations.xml